docs/sources/configuration/integrations/gcp-exporter-config.md

```diff
@@ -0,0 +1,186 @@
+---
+aliases:
+- /docs/agent/latest/configuration/integrations/gcp-exporter-config/
+title: gcp_exporter_config
+---
+
+# gcp_exporter_config
+
+## Overview
+The `gcp_exporter_config` block configures the `gcp_exporter` integration, which is an embedded version of
+[`stackdriver_exporter`](https://github.com/prometheus-community/stackdriver_exporter). This allows for the collection of 
+metrics data from [GCP Cloud Monitoring (formerly stackdriver)](https://cloud.google.com/monitoring/docs).
+
+The exporter supports all metrics available via [GCP's monitoring API](https://cloud.google.com/monitoring/api/metrics_gcp). Metrics
+exporter by this integration will follow the template `stackdriver_<monitored_resource>_<metric_type_prefix>_<metric_type>`. 
+
+The following example shows a load balancing metric:
+
+![gcp-exporter-config-metric-example](https://grafana.com/media/docs/agent/gcp-exporter-config-metric-example.png)
+
+The following list shows its attributes: \
+monitored_resource = `https_lb_rule`\
```

Comment:
Why are these `\` used for? github rendering is not very happy about it

Reply:
That's weird github is rendering it for me without issue,
![image](https://user-images.githubusercontent.com/4571540/216320165-caaaba60-0fb4-486b-92f6-e49b4915a237.png)
The `\`'s created a newline without a paragraph break. I do need to remove the `\` from line 24 though